registry/server/templates/services/renderTemplate.ts

```diff
@@ -29,6 +30,10 @@ type RenderTemplateResult = {
 };
 
 async function renderTemplate(template: string): Promise<RenderTemplateResult> {
+    if (!isTemplateValid(template)) {
+        throw new Error('HTML template has invalid structure');
```

Comment:
please add template name to the error

Reply:
Here it is problematic as in some cases it is not possible to get the template name in the context of execution.
For instance, this method is used in the update template route, where the renderTemplate function is called in the validator, and the template name is not available there.

Reply:
understood, could you please then add into that method log that template is going to be updated, with template details

Reply:
I thought that we could actually catch and wrap this error, so it could log that the template "name" wasn't created/updated and the cause error is "Invalid html". 
What do you think?

Reply:
yeap, good option. Works as well